### ISSUE TITLE
A basic app with a single button has five semantic nodes

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#2476]
It should have one. Maybe two, one for the whole screen, one for the button.

See the debugging.md file for a sample app that shows this.

**Comment by eseidelGoogle** [comment]
I'm not sure how useful this bug is separately.  When we look at semantics closely this and many other AX issues will get worked out I'm sure. :)  Removing regression keyword.

**Comment by goderbauer** [comment]
This is fixed with the new a11y algorithm and a test is in place to prevent regression: https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/simple_semantics_test.dart#L13
